src/Altinn.App.Api/Infrastructure/Telemetry/IdentityTelemetryFilter.cs

```diff
@@ -42,17 +43,17 @@ public void Process(ITelemetry item)
                 int? partyId = ctx.User.GetPartyIdAsInt();
                 int authLevel = ctx.User.GetAuthenticationLevel();
 
-                request.Properties.Add("partyId", partyId.ToString());
-                request.Properties.Add("authLevel", authLevel.ToString());
+                request.Properties.Add("partyId", partyId?.ToString(CultureInfo.InvariantCulture) ?? "");
```

Comment:
This changes the telemetry property from `null` to `""`?

Reply:
[`Nullable<T>` returns `""` when there is no value](https://github.com/dotnet/runtime/blob/c897ce62f21b061f8a53f6629fea97526cbb09a8/src/libraries/System.Private.CoreLib/src/System/Nullable.cs#L67)